modules/client/examples/02-client-addressList/07-get-members.ts

```diff
@@ -8,7 +8,7 @@ import { contextParams } from "../00-client/00-context";
 const context: Context = new Context(contextParams);
 // Create a plugin context from the simple context
 const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
-// Create an ERC20 client
+// Create an Token client
```

Comment:
This got renamed because it was copied and pasted from the tokenVoting examples and I forgot to change it from erc20 to address List, can you please change it